apps/web/src/pages/api/feedback.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-case-declarations */
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const webhooks = [
+  process.env.FEEDBACK_WEBHOOK_URL ??
+    "https://discord.com/api/webhooks/abc/123",
+];
+
+const rateEmoji = new Map([
+  ["bad", "🙁"],
+  ["meh", "😐"],
+  ["nice", "🙂"],
+]);
+
+export default async function (req: NextApiRequest, res: NextApiResponse) {
+  const { message, rate, metadata } = req.body;
+  const method = req.method;
+
+  try {
+    switch (method) {
+      case "POST":
+        const text = `New Feedback ✨
+
+Message: ${message ? message : "-"}
+Rate: ${rate ? rateEmoji.get(rate) : "-"}
```

Comment:
Should we make 'rate' mandatory?

Reply:
This needs to be configured on the component. And there is no configuration for that allowed. To include this we should fork and make our own version of the widget.